modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -701,3 +702,20 @@ will have the following visual differences compared to the standard `Modal`:
 
 Additionally, users interacting with `Modal` on touch will not be able to exit out of the modal by
 tapping on the overlay.
+
+---
+
+### Popups
+
+We've updated all popup `CloseIcon` and `CloseButton` to default to `type=button`. We did this so
+these close buttons did not submit the forms unintentionally. Without this change, you have to
+manually add `type="button"` to these components if you wrap popup contents in a `form` element.
+While this is a very unlikely that a `CloseButton` was intentionally used as an implicit submit
+button, it is still a breaking change. This change effects the following components:
```

Comment:
Any MDN docs to reference here that might help other developers?

Reply:
Default `type` for buttons: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type

React has a very small amount of documentation here: https://reactjs.org/docs/forms.html

There isn't much documentation on how forms work.

Reply:
At least the React documentation shows:
```jsx
<form onSubmit={handleSubmit}>
  <input .../>
  <button type="submit">Submit</button>
</form>
```

Most of the code I've seen in the wild does this instead:
```jsx
const handleKeyDown = e => {
  // handles the "enter" key submitting the form, but it isn't perfect... The enter key doesn't always submit
  // the form. A select, for example. The enter key when choosing a select option should close the select menu,
  // but not submit the form. The enter key should only submit the form if the select menu is closed AND the
  // `Select` has focus
  if (e.key === "Enter") {
    handleSubmit()
  }
}

<div onKeyDown={handleKeyDown}>
  <input ... />
  <button onClick={handleSubmit}>Submit</button>
  <button onClick={handleCancel}>Cancel</button>
</div>

Reply:
Should this be part of our documentation as in the links or od you think this is sufficient enough?